pkgs/top-level/all-packages.nix

```diff
@@ -10531,7 +10531,7 @@ with pkgs;
     spidermonkey = spidermonkey_1_8_5;
     python = python27;
     sphinx = python27Packages.sphinx;
-    erlang = erlangR16;
+    erlang = erlangR17;
```

Comment:
Can't build against R18. The internal build script throws an error if it detects an Erlang version higher than 17